app/controllers/api/v2/base_controller.rb

```diff
@@ -2,6 +2,7 @@ module Api
   module V2
     class BaseController < Api::BaseController
       include Api::Version2
+      include ApplicationHelper
```

Comment:
Do we really need this?

Reply:
I can't use `authorized_for` without `include`-ing it here.
If you know a better way to use helpers in the controller, I would love to hear.

Thanks

Reply:
I've missed this, if this is the impact, we should either isolate authorized_for to a separate helper or move the logic elsewhere

Reply:
what about instead of using `authorized_for` just doing:

```ruby
Authorizable.new(User.current).can?(:create_model)
```

I believe that's what `authorized_for` is doing when it's called in line 70.

Reply:
@ares @timogoebel can you review this again. I did what @ares suggested but I am not sure I did it correctly.